core/controllers/base_test.py

```diff
@@ -522,6 +522,11 @@ def test_every_method_has_decorator(self):
                     handler.__name__ == 'Error404Handler'):
                 continue
 
+            # Following handler is present in editor.py where acl_decorators
```

Comment:
Why can't editor.py import acl_decorators?

Reply:
i guess 'EditorLogoutHandler' in editor.py inherits from 'base.LogoutPage' and doesn't have get, post... handlers. so it was failing tests so it may not be required to apply acl_decorators for 'EditorLogoutHandler' also it does not need to inherit from base.BaseHandler.

Reply:
yes, acl_decorators should be applied. i have applied acl_decorators now, to check if it's valid exp_id.